app/helpers/record_helper.rb

```diff
@@ -40,7 +40,10 @@ def can_associate_granule?(granule_record, collection_state)
     # May 7 2020: Added status 'closed', this is a temporary change to allow users to associate
     # closed records until arc team either finishes reviewing all legacy imported records or
     # we fix the bug that is preventing users from marking all colors for a record.
-    return [true, nil] if %w(open in_arc_review closed).include?(collection_state)
+    # return [true, nil] if %w(open in_arc_review closed).include?(collection_state)
+
+    #  turn back on the validation of closed records to ensure integrity.
+    return [true, nil] if %w(open in_arc_review).include?(collection_state)
```

Comment:
If we don't have a test already for this, I think we should to verify it works.

Reply:
We should remove the comments as well.

Reply:
Done!